dask/tests/test_base.py

```diff
@@ -955,6 +955,28 @@ def test_visualize():
         assert os.path.exists(os.path.join(d, "mydask.png"))
 
 
+@pytest.mark.skipif("not da")
+@pytest.mark.skipif(
+    sys.flags.optimize, reason="graphviz exception with Python -OO flag"
+)
+def test_visualize_no_filename(monkeypatch):
+    pytest.importorskip("graphviz")
+    pytest.importorskip("dask.dot")
+
+    x = da.arange(10)
+
+    test_list = []
+    _mock_get_display_cls = dask.dot._get_display_cls
+
+    def mock_fx(format):
```

Comment:
As @GenevieveBuckley  said, a comment here for what we're doing (this patch ensures that display_cls got called successfully) would be good.

Reply:
Done!